config/wsgi.py

```diff
@@ -13,6 +13,6 @@
 
 from django.core.wsgi import get_wsgi_application
 
-#os.environ.setdefault("DJANGO_SETTINGS_MODULE", "config.settings.production")
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "config.settings.production")
```

Comment:
we can have this in entrypoint  file instead.....
this will always set the module to production for local dev also